grpc/src/main/java/io/stargate/grpc/service/Service.java

```diff
@@ -77,12 +84,18 @@
 import org.apache.cassandra.stargate.exceptions.WriteFailureException;
 import org.apache.cassandra.stargate.exceptions.WriteTimeoutException;
 import org.immutables.value.Value;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class Service extends io.stargate.proto.StargateGrpc.StargateImplBase {
-
+  private static final Logger log = LoggerFactory.getLogger(Service.class);
   public static final Context.Key<AuthenticationSubject> AUTHENTICATION_KEY =
       Context.key("authentication");
   public static final Context.Key<SocketAddress> REMOTE_ADDRESS_KEY = Context.key("remoteAddress");
+  private static final String SYSTEM_TRACES_KEYSPACE = "system_traces";
```

Comment:
minor:  Seem unnecessary: `private static final String SYSTEM_TRACES_KEYSPACE = "system_traces";`  It's used one time and it's very unlikely keyspace name will ever change.